src/lib/ThumbnailsSidebar.js

```diff
@@ -9,6 +9,7 @@ const CLASS_BOX_PREVIEW_THUMBNAIL_IMAGE_LOADED = 'bp-thumbnail-image-loaded';
 const CLASS_BOX_PREVIEW_THUMBNAIL_IS_SELECTED = 'bp-thumbnail-is-selected';
 const CLASS_BOX_PREVIEW_THUMBNAIL_PAGE_NUMBER = 'bp-thumbnail-page-number';
 export const DEFAULT_THUMBNAILS_SIDEBAR_WIDTH = 154; // 225px sidebar width - 25px margin right, - 40px for page number - 6px for border
+const HIGHER_RES_WIDTH = DEFAULT_THUMBNAILS_SIDEBAR_WIDTH * 4;
```

Comment:
nit: once this change goes in this will be our default resolution, should we call it something like thumbnail_image_resolution?

Reply:
I'll change it to `THUMBNAIL_IMAGE_WIDTH`